packages/web/hooks/one-click-trading/use-one-click-trading-session.ts

```diff
@@ -97,7 +100,7 @@ export const useOneClickTradingSession = ({
   return {
     oneClickTradingInfo: featureFlags.oneClickTrading ? value?.info : undefined,
     isOneClickTradingEnabled: featureFlags.oneClickTrading
-      ? value?.isEnabled
+      ? value?.isEnabled ?? false
```

Comment:
With the previous suggested comment, this would no longer be necessary

Reply:
122440450425